TelepresenceBot/core/src/main/java/com/novoda/tpbot/bot/BotPresenter.java

```diff
@@ -30,6 +30,7 @@ void stopPresenting() {
         unsubscribe(connectionObservable);
         unsubscribe(directionObservable);
         tpService.disconnect();
+        botView.onDisconnect();
```

Comment:
When disconnecting locally, performing a stop of the activity, we need to reset the views.